api/test/tree-shaking/tree-shaking.test.ts

```diff
@@ -24,97 +24,102 @@ import * as realFs from 'fs';
 /**
  * Verify that tree-shaking can be properly applied on the @opentelemetry/api package.
  * Unused optional apis should be able to be removed from the final bundle.
+ *
+ * Webpack doesn't run in node 8 because it requires BigInt. Since we are testing
+ * build tooling here, we can safely skip tooling we know can't run anyway.
  */
-describe('tree-shaking', () => {
-  const allowedAPIs = ['ContextAPI', 'DiagAPI'];
-  const testAPIs = [
-    {
-      name: 'MetricsAPI',
-      export: 'metrics',
-    },
-    {
-      name: 'PropagationAPI',
-      export: 'propagation',
-    },
-    {
-      name: 'TraceAPI',
-      export: 'trace',
-    },
-  ];
-  const APIMatcher = /(?:class|function) (\w+API)/g;
+if (parseInt(process.version.split('.')[0], 10) >= 10) {
```

Comment:
nit: we can revert the if-condition and return early to avoid indenting the file contents below.

```suggestion
if (parseInt(process.version.split('.')[0], 10) < 10) {
  process.exit();
}
```

Reply:
Hmm, IIRC mocha does only starts one process for multiple test files. :thinking: 

I'm worried that we may be exiting before these other tests run. Maybe we could use mocha's [`this.skip()`](https://mochajs.org/#inclusive-tests) (only works when not using arrow functions in `describe`/`it`, see https://mochajs.org/#arrow-functions)